Code/GraphMol/ChemTransforms/MolFragmenter.cpp

```diff
@@ -409,6 +409,25 @@ void checkChiralityPostMove(const ROMol &mol, const Atom *oAt, Atom *nAt,
     nAt->invertChirality();
   }
 }
+
+std::vector<std::pair<Bond*, std::vector<int>>> getNbrBondStereo(RWMol &mol, const Bond *bnd) {
+    // loop over neighboring double bonds and remove their stereo atom
+    std::vector<std::pair<Bond*, std::vector<int>>> res;
+    Atom *bgn = bnd->getBeginAtom();
+    Atom *end = bnd->getEndAtom();
+    for(auto *atom : {bgn, end}) {
+        ROMol::OEDGE_ITER a1, a2;
+        for (boost::tie(a1, a2) = mol.getAtomBonds(atom); a1 != a2; a1++) {
```

Comment:
```suggestion
        for (boost::tie(a1, a2) = mol.getAtomBonds(atom); a1 != a2; ++a1) {
```

Reply:
I have no idea why I default to a++ these days.